libp2p/io/msgio.py

```diff
@@ -0,0 +1,24 @@
+from libp2p.network.connection.raw_connection_interface import IRawConnection
+
+from .exceptions import MissingLengthException, MissingMessageException
+
+SIZE_LEN_BYTES = 4
+
+# TODO unify w/ https://github.com/libp2p/py-libp2p/blob/1aed52856f56a4b791696bbcbac31b5f9c2e88c9/libp2p/utils.py#L85-L99  # noqa: E501
```

Comment:
Would we want it to be unified with https://github.com/libp2p/py-libp2p/blob/1aed52856f56a4b791696bbcbac31b5f9c2e88c9/libp2p/utils.py#L85-L99 here?

Reply:
we do -- i'll get to this in a future PR tho -- not sure which approach (having flat utility functions vs working towards specialized reader/writer types) i think is nicer atm